src/ValueFormatters/TimeFormatter.php

```diff
@@ -58,17 +53,12 @@ public function format( $value ) {
 				. 'instances of DataValues\TimeValue' );
 		}
 
-		$formatted = $value->getTime();
-
 		$isoFormatter = $this->getOption( self::OPT_TIME_ISO_FORMATTER );
 		if ( $isoFormatter instanceof ValueFormatter ) {
-			$formatted = $isoFormatter->format( $value );
+			return $isoFormatter->format( $value );
 		}
```

Comment:
This code is so useless now it should just be removed. If you want to actually use another formatter, then just use it instead of constructing this formatter and passing the one you want to use in.